howso/utilities/installation_verification.py

```diff
@@ -908,7 +904,7 @@ def check_engine_operation(
     finally:
         try:
             if engine and trainee:
-                engine.delete_trainee(trainee.id)
+                engine.delete_trainee(trainee['id'])
```

Comment:
This is an engine.Trainee.

```suggestion
                engine.delete_trainee(trainee.id)
```

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60